components/config/definition.rst

```diff
@@ -310,6 +313,12 @@ The output configuration will be exactly the same as before. In other words, the
 ``sf_connection`` and ``default`` configuration keys are lost. The reason is that
 the Symfony Config component treats arrays as lists by default.
 
+.. note::
+
+    As of writing this, there is an inconsistency: if only one file is processed,
```

Comment:
even when multiple files are processed (which happens in all Symfony projects, as nobody puts the whole configuration if ``config_prod`` or ``config_dev``), as long as only 1 file is providing config for this list